.github/workflows/release-publish.yml

```diff
@@ -0,0 +1,63 @@
+name: release-publish
+
+on:
+  workflow_call:
+    inputs:
+      slack_channel:
+        required: false
+        type: string
+    secrets:
+      artifactory-api-key:
+        required: true
+      release-github-token:
+        required: true
+      slack-api-token:
+        required: true
+      binaries-aws-deploy:
+        required: true
+      binaries-aws-access-key-id:
+        required: true
+      binaries-aws-secret-access-key:
+        required: true
+      binaries-aws-region:
+        required: true
+      releasability-aws-access-key-id:
+        required: true
+      releasability-aws-secret-access-key:
+        required: true
```

Comment:
Seems that it should be possible to use one cross account. This allows us to rely on one odic authentication flow.
https://docs.aws.amazon.com/IAM/latest/UserGuide/tutorial_cross-account-with-roles.html

Reply:
Out of scope. This will be addressed later on with the ticket: [BUILD-1475](https://jira.sonarsource.com/browse/BUILD-1475)